prow/crier/reporters/gerrit/reporter.go

```diff
@@ -283,7 +287,9 @@ func (c *Client) Report(ctx context.Context, logger *logrus.Entry, pj *v1.ProwJo
 			toReportJobs = append(toReportJobs, pjOnRevisionWithSameLabel)
 		}
 	}
-	report := GenerateReport(toReportJobs)
+	// The default size limit of 5000 is way too much, will be barely readable
+	// this point, let's not use all of them
+	report := GenerateReport(toReportJobs, defaultCommentSizeLimit*4/5)
```

Comment:
style nit: *4/5 spacing inconsistent with code on line 391. 

any particular reason behind the 80% used? why not 70% or 90%? a comment on that might be helpful

Reply:
This should just be default, changed to 0 instead. Added a comment above line 391 explaining 80%

Reply:
thank you!